documentation/Database-Schema-User.md

```diff
@@ -0,0 +1,42 @@
+# Database Schema User
+
+![Database Schema](images/database-schema-user.png)
+
+# Database Area description: User
+
+## User instance
+
+An user is either the Applicant that wants to apply to a application (Registration) in the system or the other entities (i.e. Reviewer) that will be the evaluator of the workflow of the application. What will define exact the type of user are permissions (permission_join) which is associated with the user and the organisation of the user. So the same user may have different actions available depening on the company selected on the login.
+
+### user
+
+The `username` is what the user will use to login to the system and the name that show on the User area.
+
+The `password` is the secret used by the user to login - which should be stored using some encryption to be defined.
+
+The `email` might be also used to login the user to the system. The email is confirmed before the account is activated.
+
+**To be considered:**
+
+- A few more fields will be required in the user table, to be added as needed.
+
+### organisation
+
+The `name` is how the company/organisation is defined in the system.
+
+The `license_number` is a number generated by the system and stored in the organisation. It shoudl also have a expiry.
```

Comment:
Yes. Indeed. I have changed on `04_organisation.sql` file.

I think we will probably need more than this to be stored - for example the `application_id` where all the details for a company approved is stored. But I think this can be added when we get to the Epic to add the "Create company" application.